doc/ref_model/chapters/chapter10.md

```diff
@@ -25,7 +25,7 @@ As the first component, the reference model must also be structured and iterated
 
 Going forward, this chapter shall be dedicated to identifying the challenges and gaps found in the course the development--to ensure the reference model is adding the strategic and tactical value intended over time. Should a challenge or gap not be identified and it is not already addressed in the model itself, the community may assume it will remain an unknown and, therefore, the community is welcomed to engage with and raise an issue with the appropriate working group(s) to close the gap. In this manner, the Reference Model can continuously improve.
 
-For Reference - Common NFVI Lifecycle Framework Diagram (Figure 10-1)
+For Reference - Common Cloud Infrastructure Lifecycle Framework Diagram (Figure 10-1)
 <p align="center"><img src="../figures/ch10-fig-10-1-common-nfvi_lifecycle_framework.png" alt="Framework" title="Framework" width="65%"/></p>
```

Comment:
Please fix the title in the image. It still uses the NFVI term.

Reply:
The source file is missing from the artefacts folder. @jbeltranatl, @mf4716, @pgoyal01, could you please send it to us?